client/src/components/geneExpression/infoPanel/geneInfo/index.tsx

```diff
@@ -0,0 +1,161 @@
+    infoError,
+    showWarningBanner,
+  } = props;
+
+  let synonymList;
+  if (geneSynonyms.length > 1) {
+    synonymList = geneSynonyms.join(", ");
+  } else if (geneSynonyms.length === 1) {
+    synonymList = geneSynonyms[0];
+  } else {
+    synonymList = null;
+  }
+
+  console.log(gene);
+
+  return (
+    <GeneInfoWrapper id="geneinfo_wrapper" data-testid={`${gene}:gene-info`}>
+      <GeneInfoContainer id="gene-info">
+        {geneName === "" && infoError === null && gene !== null ? (
+          <GeneInfoEmpty>
+            <GeneSymbol>{gene}</GeneSymbol>
+            <Content>loading...</Content>
+          </GeneInfoEmpty>
+        ) : null}
+        {gene === null && infoError === null ? (
+          <GeneInfoEmpty>
+            <NoGeneSelectedDiv>
+              <CustomIcon icon="search" size={33} />
+
+              <MessageDiv className="title">No Gene Selected</MessageDiv>
+              <MessageDiv>
+                Choose a gene above or search the NCBI database.
+              </MessageDiv>
+            </NoGeneSelectedDiv>
+          </GeneInfoEmpty>
+        ) : null}
+        {infoError !== null ? (
+          <GeneInfoEmpty>
+            <GeneSymbol>{gene}</GeneSymbol>
+            <Content>Sorry, this gene could not be found on NCBI.</Content>
+            <Link
+              href={`https://www.google.com/search?q=${gene}`}
+              target="_blank"
+              rel="noreferrer noopener"
+            >
+              Search on Google
+            </Link>
+          </GeneInfoEmpty>
+        ) : null}
+        {geneName !== "" && infoError === null ? (
+          <GeneInfoEmpty>
+            {showWarningBanner ? (
+              <WarningBanner>
+                <Icon
+                  sdsIcon="exclamationMarkCircle"
+                  sdsSize="l"
+                  sdsType="static"
+                />
+                <span>
+                  NCBI didn&apos;t return an exact match for this gene.
+                </span>
+              </WarningBanner>
+            ) : null}
+            <GeneSymbol data-testid="gene-info-symbol">{gene}</GeneSymbol>
+            <Content data-testid="gene-info-name">{geneName}</Content>
+            {geneSummary === "" ? (
+              <Content
+                style={{
+                  display: "-webkit-box",
+                  WebkitLineClamp: "7",
+                  WebkitBoxOrient: "vertical",
+                  overflow: "hidden",
+                }}
+              >
+                This gene does not currently have a summary in NCBI.
+              </Content>
+            ) : (
+              <Content
+                style={{
+                  display: "-webkit-box",
+                  WebkitLineClamp: "7",
+                  WebkitBoxOrient: "vertical",
+                  overflow: "hidden",
+                }}
+                data-testid="gene-info-summary"
+              >
+                {geneSummary}
+              </Content>
+            )}
```

Comment:
wonder if there's opportunities to keeping the code DRY by reusing the same styling object?